LatestTagDocs/RevenueCat.doccarchive/_redirects

```diff
@@ -0,0 +1 @@
+/* /index.html 200
```

Comment:
this one is required for deploying Docc. 

Also, files from here on are auto-generated, other than the Fastfile at the very bottom of this diff